cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -70,7 +82,7 @@ def node_data
         unless quiet?
           grid_health = grid_health(grid, grid_nodes)
           grid_nodes.each do |node|
-            node['name'] = health_icon(node_health(node, grid_health)) + " " + (node['name'] || '(initializing)')
+            node['name'] = health_icon(node_health(node, grid_health)) + " " + (node['name'] || node['node_id'])
```

Comment:
This really shouldn't be necessary anymore after #2694 already fixing #2558, but it doesn't hurt either.